pkg/network/driver/netlink/netlink.go

```diff
@@ -0,0 +1,22 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2022 Red Hat, Inc.
+ *
+ */
+
+package netlink
+
+type NetLinkH struct{}
```

Comment:
What does the `H` suffix stand for?

Reply:
"Handler".
It may be fine to remove it, I must admit I do not remember why I decided to add it to all drivers.
I do not think it has any drawbacks, and for now it does not bother me much when reading it. If it will, it should be pretty easy to change.